front/src/GameplayScreen/Utils/axiosFuncs.js

```diff
@@ -0,0 +1,64 @@
+import Axios from 'axios';
+
+const axios = Axios.create({ baseURL: 'http://localhost:8888' });
+
+export const apiCallsHandler = async ({ action, data }) => {
+	const response = await calls[action](data);
+	return response.data;
+};
+
+const logError = error => {
+	console.error(`Error message: ${error.response.data}`);
+};
+
+const calls = {
+	postGame: async gameState => {
+		try {
+			return await axios.post('/api/game', {
+				boardState: gameState.boardState,
+				turnState: gameState.turnState,
+				winState: gameState.winState,
+			});
+		} catch (error) {
+			logError(error);
+			return error;
+		}
```

Comment:
.then - look at axios middleware
.catch - move catch blocks to `apiCallsHandler`